src/locales/zh/common.flt

```diff
@@ -11,7 +11,7 @@ serial-comma = {$prev}、{$next}
 -developer-wiki = 开发者 Wiki
 -forum = 论坛
 -import-utility = Quilt 导入工具
--openings = 空缺职位
+-help-wanted = Help Wanted (FIXME BEFORE MERGE!!!)
```

Comment:
TBH the Chinese translation is there just as a way for me to demonstrate the language switch. I'm not planning to make a full translation so you can leave that as-is for now :p